awscli/examples/s3/cp.rst

```diff
@@ -151,16 +151,21 @@ Output::
 
 **Uploading a local file stream to S3**
 
-WARNING:: PowerShell may alter the encoding of or add a CRLF to piped input.
+.. WARNING:: PowerShell may alter the encoding of or add a CRLF to piped input.
 
 The following ``cp`` command uploads a local file stream from standard input to a specified bucket and key::
 
     aws s3 cp - s3://mybucket/stream.txt
 
+**Uploading a local file stream that is larger than 50gb to S3**
```

Comment:
Let's make the casing here `50GB` to indicate gigabytes. If it is lowercase, users may think it is in gigabits.